app/src/main/java/com/jerboa/ui/components/login/Login.kt

```diff
@@ -150,11 +161,23 @@ fun LoginForm(
             }
         }
         MyTextField(
+            modifier = Modifier
+                .background(if (wasAutofilled) Color(0xFFFFFDE7) else Color.Transparent)
```

Comment:
No hardcoding colors like that. Use material 3 colors.

Reply:
I couldn't find any color for this so I'm not sure what the proper way to this would be. Maybe adding an extension to `ColorScheme`?

Reply:
I'm sure one of these would work, no need to add extensions.

https://m3.material.io/styles/color/the-color-system/color-roles

Try different ones out and and see what looks good.

`MaterialTheme.colorScheme...`